README.md

```diff
@@ -41,3 +41,35 @@ If you're interested in examples of a specific version of the API, they can be f
 
 [Protecting Your First App]: https://docs.authzed.com/guides/first-app
 [examples directory]: /examples
+
+### Getting Started
+
+```js
+const authzed = require('@authzed/authzed-node/v0');
+const client = new authzed.Client("yourtokenhere");
+
+// Make a Check request.
+const resourceAndPermission = new authzed.ObjectAndRelation();
+resourceAndPermission.setNamespace("mytenant/note");
+resourceAndPermission.setObjectId("README");
+resourceAndPermission.setRelation("reader");
+
+const subjectReference = new authzed.ObjectAndRelation();
+subjectReference.setNamespace("mytenant/user");
+subjectReference.setObjectId("jimmy");
+subjectReference.setRelation("...");
+
+const user = new authzed.User();
+user.setUserset(subjectReference);
+
+const checkRequest = new authzed.CheckRequest();
+request.setTestUserset(resourceAndPermission);
+request.setUser(user);
+
+client.acl.check(checkRequest, function (err, response) {
+    console.log(response);
+    console.log(err);
+});
```

Comment:
Let's make this just show off a check call and not a full example and move it into the examples ection